index.html

```diff
@@ -214,11 +216,49 @@ <h1 class="wrapper">About me</h1>
     </section>
       
     <!-- About End -->
-      
+
+    <!-- Contact Start -->
+
+    <section class="wrapper">
+      <div class="contact">
+        <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+        <form action="https://formspree.io/f/xqkovylg" method="post">
+          <label>
+            <input type="text" name="name" placeholder="Full name" maxlength="30" required />
+          </label>
+          <label>
+            <input type="email" name="email" placeholder="Email address" required />
+          </label>
+          <label>
+            <textarea name="message" placeholder="Write me something..." maxlength="500" required></textarea>
+          </label>
+          <button class="button" type="submit">Get in touch</button>
```

Comment:
Great UI! 👌🎨 Your interface is looking fantastic, and you've done an excellent job overall. Currently, your button has styles for the enabled and hover states, which is great. But as per the design specifications, it's important to also include styles for the active and disabled states. 🎨⚡

I kindly suggest adding the necessary styles for the active and disabled states of the button. This will ensure a consistent and visually appealing experience for users, aligning with the design requirements. ✨🔘
![image](https://github.com/mahammad-mostafa/personal-portfolio-site/assets/55840999/8db9daa6-c3cd-44a8-b814-65c74fcef619)

Keep up the fantastic work! 💪🤗

Reply:
Hi @shahadat3669,
Thanks for your feedback. I will update the README file to include my partner in the authors section.
However, I have already implemented the disabled and pressed states.
I just do not have disabled buttons on the page. But you can check the states in my CSS file.
Looking forward to your feedback,

Reply:
@mahammad-mostafa that's good.